### ISSUE TITLE
Question about  VAE training

### ISSUE THREAD

**Issue by XiangL-Xr** [monkek123King/CLD#5]
Hi, thank you for the great work and for making the code available. I am trying to reproduce the results and to gain a deeper understanding of the method proposed in the paper. However, I encountered some problems when trying to reproduce the Transparent VAE results. Following the method described in the paper, I trained the VAE's decoder using L1 loss, and I also tried adding perceptual loss. However, the images decoded during inference always showed mosaic-like pixels. Therefore, I would like to ask if you have encountered similar problems when training Transparent VAE, or if there are any other details I should pay attention to. Thank you for your time！

**Comment by monkek123King** [comment]
Hi, thanks for your question.

In our work, the Transparent VAE is directly taken from ART with their released pretrained weights, and we do not train or finetune the VAE in our pipeline. Therefore, we did not encounter training-related issues such as mosaic artifacts during decoding.

If you are training the Transparent VAE from scratch, the mosaic-like artifacts are likely related to the VAE training setup itself. For details on the VAE architecture and training strategy, we recommend referring to the original ART paper and implementation:
https://arxiv.org/abs/2502.18364

Hope this clarifies things.

**Comment by XiangL-Xr** [comment]
Thanks for your reply！
